tools/src/Npm.ts

```diff
@@ -82,6 +82,7 @@ export async function publishPackageAsync(
   }
   await spawnAsync('npm', args, {
     cwd: packageDir,
+    stdio: 'inherit',
```

Comment:
I think it would be better not to inherit the stdout as it will produce a lot of unnecessary bloating logs.

Reply:
But how would we get the OTP link then?

Reply:
And that's the problem with proper implementation that supports OTP 😅 I'm a bit reluctant to bloat the logs just to support this. You would have to use `pipe` and parse the stream to detect the link.

Reply:
Looks like there is `npm profile get --json` which returns an object with user's settings. We can read `tfa.mode` that is `auth-and-writes` when an OTP is needed for publishes 🙂

Reply:
Nice, that makes things way simpler, I've just updated the function to check for that in order to decide which `stdio` to use